lib/commands/element.js

```diff
@@ -162,12 +162,15 @@ commands.setValue = async function (value, el) {
     //   ['some text']
     //   ['s', 'o', 'm', 'e', ' ', 't', 'e', 'x', 't']
     //   'some text'
-    if (_.isString(value)) {
-      // plain string, so make it into an array of characters
-      value = value.split('');
-    } else if (_.isArray(value)) {
+    if (_.isArray(value)) {
       // make sure that all the strings inside are a single character long
       value = _.flatMap(value, (v) => (_.isString(v) ? v : JSON.stringify(v)).split(''));
+    } else {
+      if (!_.isString(value)) {
+        value = value.toString();
```

Comment:
I'm not convinced that when `null` is passed in `'null'` ought to be the value set.